package.json

```diff
@@ -4,10 +4,45 @@
   "version": "5.0.0",
   "description": "Turf.js documentation",
   "scripts": {
-    "build": "node scripts/create-config.js",
-    "postbuild": "webpack --progress --hide-modules",
-    "start": "webpack-dev-server --open --hot",
-    "lint": "eslint --ext .js,.vue src"
+    "create-config": "node scripts/create-config.js",
+    "start": "nuxt",
```

Comment:
@rowanwins I dropped `start & build` in favor of using the "hot reloading" `nuxt` command.

From what I gather the `nuxt generate` command builds the bundle for the web, so no need to run `nuxt build` & `nuxt start`